app/models/case_contact.rb

```diff
@@ -33,13 +33,13 @@ class CaseContact < ApplicationRecord
     where("occurred_at BETWEEN ? AND ?", start_date, end_date) if start_date.present? && end_date.present?
   }
   scope :contact_made, ->(contact_made = nil) {
-    where(contact_made: contact_made) if contact_made == true || contact_made == false
+    where(contact_made: contact_made == "true") if contact_made == "true" || contact_made == "false"
```

Comment:
Is there a reason to do this vs converting the string to a boolean earlier? This is making the code pretty confusing.

Reply:
I found this: https://stackoverflow.com/questions/19339137/how-to-set-boolean-values-to-radio-buttons-in-rails-4-form

Reply:
I agree with you, I tried changing the radio button to a boolean value but without success. I'm going to try again this way you linked, brb in a couple of minutes.

Reply:
I tried this way and the params are still strings, after some research I think its the intended design of radio buttons to pass the value as strings.

the snippet I used:
```ruby
  <div class="field form-group">
    <label><strong>Transition Aged Youth:</strong></label>
    <div class="form-check">
      <%= f.radio_button(:has_transitioned, '', class: "form-check-input", checked: true, value: '') %>
      <%= f.label(:has_transitioned, 'Both', class: "form-check-label") %><br>

      <%= f.radio_button(:has_transitioned, true, class: "form-check-input", value: true) %>
      <%= f.label(:has_transitioned, 'Yes', class: "form-check-label") %><br>

      <%= f.radio_button :has_transitioned, false, value: false, class: "form-check-input" %>
      <%= f.label :has_transitioned, "No", value: false, class: "form-check-label" %><br>
    </div>
  </div>
```
params passed
![image](https://user-images.githubusercontent.com/9721558/97227498-5571b700-17b4-11eb-98cd-13a28cb3de63.png)

Reply:
Yeah I'm also in favor of just converting after the strong params. Also, is there a reason to have both? Either a youth has or hasn't transitioned...